build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:4.2.0'
```

Comment:
I am sorry that I said 4.x before though I actually wanted to update it to `7.x` which is the latest and also utilised by commcare-android.

Reply:
No problem, let me correct that.